tests/percy/photographer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Responsible for navigating pages and capturing snapshots
+ */
+class Photographer {
```

Comment:
Is this JSDoc still accurate? It seems like this class is only responsible for page navigation. If that's the case, I think we should change the name from `Photographer`. Maybe something like `PageNavigator`.

Reply:
Ya, that jsdoc got out of date with a refactor. PageNavigator works!